### ISSUE TITLE
Combobox: Invalid ARIA attribute values: aria-describedby and aria-owns when disabled

### ISSUE THREAD

**Issue by random-creator** [OfficeDev/office-ui-fabric-react#4873]
### Bug Report

- __Package version(s)__: 5.91.1002
- __Browser and OS versions__: chrome

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No) yes

Requested priority: (Blocking, High, Normal, Low)  Normal

Products/sites affected: (if applicable): VSTS

#### Describe the issue:
Keros FastPass scan fails with this error on combox in disabled state without any selection.

Rule: aria-valid-attr-value:  ARIA attributes must conform to valid values
Fix the following:
Invalid ARIA attribute values: aria-describedby="some id-option" aria-owns="some id-list"

#### Actual behavior:
FastPass scan shows this error.

#### Expected behavior:

This issue doesn't show in FastPass.

Additional note:
It seems like aria-describedby and aria-owns should be undefined when the combobox is disabled with no selection.
